src/mon/PGMonitor.h

```diff
@@ -38,10 +38,13 @@ class MGetPoolStats;
 
 class TextTable;
 class MPGStats;
+class PGStatService;
+class PGMapStatService;
 
 class PGMonitor : public PaxosService {
 public:
   PGMap pg_map;
+  PGMapStatService *pgservice;
```

Comment:
@gregsfortytwo what's our plan to extricate the pgmap from mon? IIUC, you are tracking the necessary bits of pgmap used by mon using pgservice. and what's the next step? will we remove PGMonitor completely, and or we will just ditch `PGMonitor.pg_map`, and keep a minimal version of pgmap in `PGMonitor`?